notebooks/generate_synthetic_data.ipynb

```diff
@@ -0,0 +1,258 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        response = model.generate_content(prompt)
</pre><p>prompt should be passed as a parameter</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/llmops-pipeline/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>